src/resmom/stage_func.c

```diff
@@ -155,37 +155,40 @@ int
 is_child_path(char *dir, char *path)
 {
 	char fullpath[2*MAXPATHLEN+2] = {'\0'};
-	char dir_real[MAXPATHLEN+1] = {'\0'};
-	char fullpath_real[2*MAXPATHLEN+2] = {'\0'};
+	char *dir_real = NULL;
+	char *fullpath_real = NULL;
 	char *pos = NULL;
+	int  return_value = FALSE;
 
 	/* if file path is relative, combine it with directory */
 	if (!is_full_path(path)) {
-		(void)snprintf(fullpath, sizeof(fullpath)-1, "%s/%s", dir, path);
+		(void)snprintf(fullpath, sizeof(fullpath), "%s/%s", dir, path);
 	} else {
-		strncpy(fullpath, path, sizeof(fullpath)-1);
+		strncpy(fullpath, path, sizeof(fullpath));
 	}
 
 	/* even if the file path is relative to some directory, */
 	/* it may use /../../ notation and escape that directory, */
 	/* so always perform full check of parent-child directory relation */
+	dir_real = realpath(dir, NULL);
+	fullpath_real = realpath(fullpath, NULL);
 #ifdef WIN32
-	forward2back_slash(fullpath);
```

Comment:
Why did you removed this line? Is this change intentional?

Reply:
Yes this change is intentional. Windows version of `realpath` function has a conversion from back slash to forward slash. So both `dir_real` and `fullpath_real` will be in forward slash for the substring search.